src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,110 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
-
+import PropTypes from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+
+import { getMovie } from '../../api';
+
+export const FindMovie = ({ addMovie }) => {
+  const [value, setValue] = useState('');
```

Comment:
```suggestion
  const [query, setQuery] = useState('');
```
It's hard to understand for what purposes this variable is. Let's name it more meaningful

Reply:
Fixed